python/src/equistore/operations/slice.py

```diff
@@ -22,13 +24,13 @@ def slice(tensor: TensorMap, samples_to_slice=None, properties_to_slice=None):
 
         sliced_tensor = slice(
             tensor,
-            samples=Labels(
+            samples_to_slice=Labels(
```

Comment:
should this really be named `samples_to_slice`? Given we are calling the `slice` function, this feels redundant :smiley:

Reply:
Good point! I can do the reverse, but it will be a breaking change (i.e. for the equistore examples thing that uses it)

Reply:
I'm fine with breaking equistore-example, that's kind of the point =) (figuring out an API and changing it until it is great!)